generate-stackbrew-library.sh

```diff
@@ -11,7 +11,9 @@ array_10='10 dubnium'
 # shellcheck disable=SC2034
 array_12='12 erbium lts current'
```

Comment:
why is 12 `current`? do we mean it's `current lts`? `current` in node terms is equivalent to `latest` in docker terms

Reply:
@PeterDaveHello you changed this in your PR - thoughts?

Reply:
#1249